app/frontend/src/assets/scss/style.scss

```diff
@@ -114,3 +131,112 @@ a, .v-tab {
     }
   }
 }
+
+// override formio form builder
+.formbuilder,
+// override form viewer
+.form-wrapper,
+// override preview panel in form builder component editor modal
+.component-preview{
+
+  // layout
+  div[class^="col-"]:nth-child(n+1) {
+    padding-top: 0; // make nested columns line up vertically alongside components in non-nested adjacent columns
+  }
+
+  // form inputs
+  .form-control {
+    border: 1px solid #606060;
+  }
+
+  // buttons
+  .btn {
+    @extend .v-btn;
+  }
+
+  .btn-primary {
+    //@extend .primary; // unable to extend selectors in other stylesheets (??)
+    background-color: $bcgov-blue;
+    border: none;
+
+    &:hover{
+        background-color: $bcgov-blue;
+        text-decoration: underline;
+    }
+
+    &:disabled,
+    &[disabled="disabled"],
+    &[disabled="disabled"] &:hover{
+      @extend .v-btn--disabled;
+      cursor: default;
+      text-decoration: none;
+      opacity: 1;
+    }
+  }
+
+  // labels
+  label:not(.form-check-label) {
+    &:after {
+      content: " (optional)";
+      color: inherit;
+    }
+
+    &.field-required:after {
+      content: "";
+    }
+  }
+
+  .tooltip-inner{
+    background-color: #606060;
+    border-radius: 3px;
+    box-shadow: 0 2px 8px 0 rgba(175, 175, 175, 0.5);
+    font-size: .85em;
+  }
+
+  // validation messages and help text
+  .form-text{
+    font-size: .85em;
+  }
+  .invalid-feedback{
```

Comment:
Make sure to use VSCode autoformatter to keep spacing consistent. I think there's supposed to be a space between feedback and { but can't remember SCSS conventions.